cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -9,6 +9,58 @@ toc::[]
 [role="_abstract"]
 You can scan images stored in image registries, including cluster local registries such as the {ocp} integrated image registry by using the `roxctl` CLI.
 
+[id="scanning-images-by-using-a-remote-cluster_{context}"]
+== Scanning images by using the Central cluster
```

Comment:
Not sure how to best message this, but IF the delegated scanning config has an entry matching the image being scanned, it will be sent to a secured cluster / not scanned by Central.